containers/jenkins-master/config/jobs/github-snapcraft-integration-tests-cloud/config.xml

```diff
@@ -0,0 +1,122 @@
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers>
+    <org.jenkinsci.plugins.ghprb.GhprbTrigger plugin="ghprb@1.29.4">
+      <spec>H/5 * * * *</spec>
+      <latestVersion>3</latestVersion>
+      <triggerPhrase></triggerPhrase>
+      <configVersion>3</configVersion>
+      <adminlist>elopio
+kyrofa
+sergiusens
+      </adminlist>
+      <allowMembersOfWhitelistedOrgsAsAdmin>false</allowMembersOfWhitelistedOrgsAsAdmin>
+      <orgslist>ubuntu-core</orgslist>
+      <cron>H/5 * * * *</cron>
+      <buildDescTemplate></buildDescTemplate>
+      <onlyTriggerPhrase>false</onlyTriggerPhrase>
+      <useGitHubHooks>true</useGitHubHooks>
+      <permitAll>false</permitAll>
+      <whitelist></whitelist>
+      <autoCloseFailedPullRequests>false</autoCloseFailedPullRequests>
+      <displayBuildErrorsOnDownstreamBuilds>false</displayBuildErrorsOnDownstreamBuilds>
+      <whiteListTargetBranches>
+        <org.jenkinsci.plugins.ghprb.GhprbBranch>
+          <branch></branch>
+        </org.jenkinsci.plugins.ghprb.GhprbBranch>
+      </whiteListTargetBranches>
+      <project>github-test</project>
+      <__project class="project" reference="../../.."/>
+      <gitHubAuthId>c166a6ec-1866-48b3-b336-d9d40f218133</gitHubAuthId>
+      <extensions>
+        <org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus>
+          <commitStatusContext>Examples tests</commitStatusContext>
+          <triggeredStatus>Triggered</triggeredStatus>
+          <startedStatus>Started</startedStatus>
+          <statusUrl></statusUrl>
+          <completedStatus>
+            <org.jenkinsci.plugins.ghprb.extensions.comments.GhprbBuildResultMessage>
+              <message>Success</message>
+              <result>SUCCESS</result>
+            </org.jenkinsci.plugins.ghprb.extensions.comments.GhprbBuildResultMessage>
+          </completedStatus>
+        </org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus>
+      </extensions>
+    </org.jenkinsci.plugins.ghprb.GhprbTrigger>
+  </triggers>
+  <concurrentBuild>true</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>#!/bin/bash
+sudo apt-get install -y build-essential bzr dpkg-dev git mercurial pyflakes python3.4 python3-apt python3-docopt python3-coverage python3-fixtures python3-jsonschema python3-mccabe python3-pep8 python3-pip python3-requests python3-testscenarios python3-testtools python3-yaml python3-lxml squashfs-tools
+
+. $HOME/.openstack/novarc
+
+if [ "$ghprbTargetBranch" = "1.x" ]; then
+  # 1.x needs a slave older than xenial.
+  exit 0
+else
+  release="rolling"
+fi
+
+snappy-cloud-client create -release $release -channel edge > instance.info
+id=$(head -n 1 instance.info | awk 'NF>1{print $NF}')
+ip=$(tail -n 1 instance.info | awk 'NF>1{print $NF}')
+
+trap "snappy-cloud-client kill $id" EXIT
+
+./runtests.sh examples --subunit --ip $ip
```

Comment:
Ok, now I understand, snappy-cloud-client comes with the latest snappy-tests-job, if you try a fresh deploy with the current tip of master it should be available there